statsmodels/distributions/quantiles_estimation.py

```diff
@@ -0,0 +1,349 @@
+# -*- coding: utf-8 -*-
+"""Using empirical quantiles to estimate loc and scale
+
+Created on Sun Aug 19 23:10:25 2012
+Author: Josef Perktold
+
+random idea for future while looking at order statistics:
+
+for qqplot, probability plot
+like the line, it would be nice to give an option to add confidence intervals
+as visual aid
+(asymptotically it doesn't look difficult, something
+like http://en.wikipedia.org/wiki/Order_statistic#Large_sample_sizes and maybe in the SAS help based on estimated distribution I guess.
+It might not be very accurate in small samples.)
+
+also with outliers providing a line estimated by RLM is better
+
+
+
+using qqplot, quantiles to estimate loc and scale for fixed shape parameters
+
+It is (or is similar to) GMM on percentiles with an identity weight matrix.
+(where's my GMM version?)
+
+References
+----------
+
+Hassanein Biometrika 1971, Percentile Estimators for the Parameters of the
+Weibull Distribution.
+    typo in formula for V^{-1}, original in Hammersley and Morton 1964
+
+Hammersley, J. M. and K. W. Morton (1964):
+The Estimation of Location and Scale Parameters from Grouped Data,
+Biometrika, Vol. 41, No. 3/4 (Dec., 1954), pp. 296-301
+Stable URL: http://www.jstor.org/stable/2332710 .
+
+
+"""
+
+import numpy as np
+from scipy import stats
+import statsmodels.api as sm
+
+
+class QuantilesDistUniform(object):
+    '''distribution of quantiles of the uniform distribution
+
+    Warning : no error checking for valid arguments.
+        p in interval [0,1] or (0,1)
+        kth in range(1,nobs)
+
+    What I need or want:
+        * theoretical distribution, expected for p
+        * distribution at all data points, kth = range(1,nobs)
+        * distribution only at certain orders for estimation
+        * estimation of loc and scale, given shape parameters
+        * compare with GMM code
+        * use for Probability Plot confidence intervals
+
+    INCOMPLETE
+
+    '''
+
+    def __init__(self, data=None, sorted=False, nobs=None):
```

Comment:
Is there a reasonable alternative name besides `sorted`?  We should avoid overlap with builtin names.